src/test/attachments.spec.js

```diff
@@ -7,19 +7,31 @@ const Document = "./fixture/attachments.json";
 const TabsItems = Selector("[data-testid='tabs'] .nav-item");
```

Comment:
Should we add a test where a file contain attachments that should be hidden?

Reply:
ok sure